benchmarks/sleeping/yamls/knative/kn-sleeping-go.yaml

```diff
@@ -0,0 +1,44 @@
+# MIT License
+
+# Copyright (c) 2022 EASE lab
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: sleeping-go
+  namespace: default
+spec:
+  template:
+    spec:
+      nodeSelector:
```

Comment:
it is to control which node the benchmark should be invoked on. This is because one of my experiments requires spinning benchmark to be invoked on a high frequency node, sleeping benchmark on a low frequency node (which is why we need them in two different knative service with different knative yaml file to specify different nodeSelector) It did not pass the CI previously, because we need to run a command to enable this feature in Knative Serving that allows us to specify a NodeSelector in the pod spec of our Knative service first before deploying the service.